src/components/DpAutocomplete/DpAutocomplete.vue

```diff
@@ -55,7 +56,7 @@ export default {
     placeholder: {
       type: String,
       required: false,
-      default: () => Translator.trans('search')
+      default: () => de.placeholder
```

Comment:
If You do things like this here, I would remove the unnecesarry method from label -> default too...
```
    label: {
      type: String,
      required: false,
      default: 'label'
    },
```

Reply:
Done in https://github.com/demos-europe/demosplan-ui/pull/396/commits/3ec68451d9f289f5e26008050ffc3ee52df18642